src/main/scala/uk/gov/hmrc/nunjucks/NunjucksHelperModule.scala

```diff
@@ -51,6 +53,35 @@ object NunjucksHelper {
 
   val className = "_nunjucksHelper"
 
+  private val routesInitScript =
```

Comment:
Would it be worth looking at potential performance implications of this new mechanism? Does this get evaluated on every page render?

Reply:
That's already the case. The performance will definitely be slightly worse, but it'll basically be no different for services with fewer than 100 routes anyway.